public/config.template

```diff
@@ -1,4 +1,4 @@
 // DevOps context
-const configUrlApi = DEVOPS_CONTEXT_PLATFORM_APP_CONFIG_URL_API
-const configUrlApiBeta = DEVOPS_CONTEXT_PLATFORM_APP_CONFIG_URL_API_BETA
-const configGTMID = DEVOPS_CONTEXT_PLATFORM_APP_GTMID
\ No newline at end of file
+var configUrlApi = DEVOPS_CONTEXT_PLATFORM_APP_CONFIG_URL_API
+var configUrlApiBeta = DEVOPS_CONTEXT_PLATFORM_APP_CONFIG_URL_API_BETA
+var configGTMID = DEVOPS_CONTEXT_PLATFORM_APP_GTMID
```

Comment:
I would use 'DEVOPS_CONTEXT_PLATFORM_APP_CONFIG_GTMID'

Reply:
These variables won't change during the lifetime of the running SPA, I was wondering whether it be possible to keep them as 'const' or they need to be changed to 'var'.

Reply:
I would change the name of the variable to something like 'configGoogleTagManagerID', and also the placeholder to 'DEVOPS_CONTEXT_PLATFORM_APP_CONFIG_GOOGLE_TAG_MANAGER_ID'. I'm aware that it's a lot longer, but will make the meaning more explicit when reading the code, the IDE will autocomplete usages anyway, and it may get minified in the future when compiling the bundle.

Reply:
They were changed to `var` to make them global and be able to access them from `src/config/Config.js` through the [global](https://developer.mozilla.org/en-US/docs/Glossary/Global_object) `window` object and avoid having to disable eslint for every new variable that we add.

Reply:
Thanks!